### PR TITLE
Handbook: instructions about the handout is incorrect #116

### PR DESCRIPTION
**Pull request by harishv7**
Fixes: #116 
Here is a preview: http://harishv7.github.io/website-1/

New description: Each lecture consists of 1-3 sections, supported by
separate handouts. The full handbook containing all the sections is
uploaded to IVLE workbin at the beginning of the semester.
You are welcome to, but not required to, read through them before
attending the lecture.

### CODE REVIEW

File: contents/handbook-lectures.html [hunk 1]
```diff
@@ -36,8 +36,7 @@
     Please treat the webcast as a 'backup' for you to catch up anything missed during the lecture.
 </p>
 <p>
-    <strong>Handouts</strong>: Each lecture consists of 1-3 sections, supported by separate handouts. 
-    These handouts will be uploaded to the module website (see the 'Schedule' page) by Wednesday midnight of the same week. 
+    <strong>Handouts</strong>: Each lecture consists of 1-3 sections, supported by separate handouts. The full handbook containing all the sections is uploaded to IVLE workbin at the beginning of the semester.
```
Comment (damithc) [inline contents/handbook-lectures.html]: Limit line length to about 110 chars. Otherwise code review requires horizontal scrolling.

**Comment by damithc** [comment]
Minor comment added.

**Comment by harishv7** [comment]
Fixed commit to ensure max of 110 chars per line

File: contents/handbook-lectures.html [hunk 2]
```diff
@@ -37,8 +37,10 @@
 </p>
 <p>
     <strong>Handouts</strong>: Each lecture consists of 1-3 sections, supported by separate handouts. 
-    These handouts will be uploaded to the module website (see the 'Schedule' page) by Wednesday midnight of the same week. 
-    You are welcome to, but not required to, read through them before attending the lecture.
+	
+	The full handbook containing all the sections is uploaded to IVLE workbin at the beginning of the semester. 
+    
+	You are welcome to, but not required to, read through them before attending the lecture.
```
Comment (damithc) [inline contents/handbook-lectures.html]: Why blank lines?

File: contents/handbook-lectures.html [hunk 3]
```diff
@@ -36,9 +36,9 @@
     Please treat the webcast as a 'backup' for you to catch up anything missed during the lecture.
 </p>
 <p>
-    <strong>Handouts</strong>: Each lecture consists of 1-3 sections, supported by separate handouts. 
-    These handouts will be uploaded to the module website (see the 'Schedule' page) by Wednesday midnight of the same week. 
-    You are welcome to, but not required to, read through them before attending the lecture.
+    <strong>Handouts</strong>: Each lecture consists of 1-3 sections, supported by separate handouts.
+	The full handbook containing all the sections is uploaded to IVLE workbin at the beginning of the semester.
+	You are welcome to, but not required to, read through them before attending the lecture.
```
Comment (damithc) [inline contents/handbook-lectures.html]: Why are these two lines indented more than the one above?

Comment (harishv7) [inline contents/handbook-lectures.html]: Added new commit. Is that fine?

**Comment by wkurniawan07** [comment]
If the text content is acceptable, then this is good to go as well.

**Comment by wkurniawan07** [comment]
I just realized you made this PR from your `master` branch. Can you make another one from a correctly named branch?

**Comment by harishv7** [comment]
Created new pull request from correct branch

**Pull request closed without merging**